### DISCUSSION TITLE
Simple little GUI I put together

### DISCUSSION THREAD

**Discussion by rainlizard** [city96/ComfyUI-GGUF#145] [Show and tell]
https://github.com/rainlizard/EasyQuantizationGUI/releases

It's nothing special, I put it together using Cursor. I'm sure you guys are much faster with command lines so it's probably useless to most of you.

I also wrote this quick guide directed at newbies who are out of the loop: https://civitai.com/articles/8322

![379740135-11d2315b-9ea4-4caf-a3a0-e211defae7a7](https://github.com/user-attachments/assets/6e941595-e7f1-4e1b-8e6e-d79e76cf4fa1)

**Comment by JorgeR81** [comment]
I tried this but I must be missing some requirement.

When I clicked the bat file, I saw the console opening, but it closed immediately.

**Reply by rainlizard** [reply]
yeah it's probably another dependency issue.

If you've got the time, redownload it - I've added a `debug.bat` which pauses the console window so it won't disappear and you can see the error, what does it say?

**Reply by JorgeR81** [reply]
Ah right ! ... it says it does not find python.

I don't have it in my system. 

I use the python_embeded from my Comfy portable install.

Maybe it would be easier to have a custom node as @city96 suggested ( https://github.com/city96/ComfyUI-GGUF/issues/141#issuecomment-2439733094 )

**Reply by rainlizard** [reply]
oh that's interesting, I thought for sure that python would've already been installed onto everyone's computer due to all this image gen stuff. I didn't know comfyui worked like that.

I tried turning the GUI into an executable but doing that balloons the filesize to 340mb (due to 'torch' apparently). So I think it's probably best to just suggest that people install python for this application. I'll add a link to https://www.python.org/downloads/ on the download page.

A custom node would definitely be preferable yeah. It's outside of my knowledge though.

**Reply by city96** [reply]
I guess you could edit your bat file and add a path to the python executable. Like how [A1111 does it](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/82a973c04367123ae98bd9abdf80d9eda9b910e2/webui.bat#L7).

```bat
@echo off

REM set to your python path, e.g. "C:\ComfyUI_windows_portable\python_embeded\python.exe"
REM leave empty for system python
set PYTHON=

if not defined PYTHON (set PYTHON=python)
%PYTHON% EasyQuantizationGUI.py
```

(And yeah, I'll try to work on a custom node sometime.)

**Comment by Kwisss** [comment]
Tried it on the new ltxv update, but I get this error:
llama_model_quantize: failed to quantize: unknown model architecture: 'ltxv'

does llama need a patch it does not apply or something?
